modules/t/hgvs_parser.t

```diff
@@ -544,9 +563,9 @@ my %test_output_no_shift = (
            "ENSP00000435307.1:p.Arg151_Ser152insTer",
             "insertion, stop gained [-1]"
            ],
-     3 =>  ["NC_000013.10:g.51519667_51519668insG",    ##rs17857128
+     3 =>  ["NC_000013.10:g.51519668dup",    ##rs17857128
```

Comment:
Remove rsID, this is not the same variant:
```suggestion
     3 =>  ["NC_000013.10:g.51519668dup",
```

Reply:
updated 👍